packages/extension-browser/src/devtools/views/pages/analyze.css

```
@@ -33,8 +33,18 @@
 .messages {
     align-items: center;
     display: grid;
-    grid-template-columns: 50% 50%;
-    width: 100%;
+    gap: 1.5rem; /* 24px */
+    margin: 1.5rem 0; /* 24px */
+}
+
+/* Horizontal layout */
+@media (min-width: 36rem /* 576px */) {
```

Also from testing, I think we can stay in the horizontal layout a bit longer before collapsing this particular piece of UI.
```suggestion
@media (min-width: 30rem /* 480px */) {
```